docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -0,0 +1,135 @@
+
+```bash
+kwil-admin migration approve <proposal-id>
+```
+
+Operators can check the status of the proposal and the votes information as shown below:
+
+```bash
+kwil-admin migration status <proposal-id>
+```
+
+:::note
+The migration proposal will get expired if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days with a 6sec block interval). The proposal can be resubmitted after the expiration.
+:::
+
+## Activation Period
+
+Once the migration proposal is approved, the migration is set to start after the activation period. During this activation period, no new migration proposals can be submitted or approved. The activation period allows all network participants to prepare for the upcoming migration.
+
+## Migration Starts
+
+The network migration begins once the specified activation period (measured in blocks) has passed since the approval of migration proposal. The following sections provide a detailed breakdown of the events that take place on both the existing and the new network throughout the migration period.
+
+### Existing Network During Migration
+
+At the start of the migration, all the existing nodes will take a network snapshot and generate the genesis configuration files that will be used to initialize the new network. From this height onwards, all the nodes on the existing network records state changes such as account spends or changes to user data on a per-block basis and replicate these changes onto the new network.
+
+Starting from this height, the existing network will continue to operate but with limited capabilities. It will continues to support "inserts/updates/deletes" on user datasets as usual but the below functionalities will be disabled during the migration period:
+- Schema Deploy and Drop
+- Account Credits and Transfers
+- Resolution Creation
+- Resolution Approvals
+
+### New Network During Migration
+
+New network can be started in parallel to the existing network with the genesis-state as the state of the existing network at the start of the migration and then replicate the state changes of the existing network to the new network during the migration period to stay in sync with the existing network.
+
+#### Replicate State Changes From Existing Network
+
+State replication from the existing network onto the new network is handled by a built-in `migrations` listener extension in the Kwil node. The `migrations` listener extension polls for the state changes on the existing network and records these changes on the new network using governance transactions. Once these transactions are approved by the new network, the changes are applied to the database. 
+
+The `migrations` listener extension can be enabled by setting the below configuration in the `config.toml` and `genesis.json` files.
```

Comment:
I think the fact that it uses a listener extension does not need to be covered in the docs. That is really an implementation detail.

All this section needs to say is something along the lines of:

"To enable validators in the new network to replicate changes from the old network, the new validators need to set the following configurations:..."